docs/source/deployorderer/ordererchecklist.md

```diff
@@ -0,0 +1,307 @@
+
+This checklist covers key configuration parameters for setting up a production ordering service. Of course, you can always refer to the orderer.yaml file for additional parameters or more information. It also provides guidance on which parameters should be overridden. The list of parameters that you need to understand and that are described in this topic include:
+
+* [General.ListenAddress](#general-listenaddress)
+* [General.ListenPort](#general-listenport)
+* [General.TLS.*](#general-tls)
+* [General.Keepalive.*](#general-keepalive)
+* [General.Cluster.*](#general-cluster)
+* [General.BoostrapMethod](#general-bootstrapmethod)
+* [General.BoostrapFile](#general-bootstrapfile)
+* [General.LocalMSPDir](#general-localmspdir)
+* [General.LocalMSPID](#general-localmspid)
+* [FileLedger.Location](#fileledger-location)
+* [Operations.*](#operations)
+* [Metrics.Provider](#metrics-provider)
+* [Metrics.Statsd.*](#metrics-statsd)
+* [Consensus.*](#consensus)
+
+## General.ListenAddress
+
+```
+# Listen address: The IP on which to bind to listen.
+ListenAddress: 127.0.0.1
+```
+
+* **`ListenAddress`**: (default value should be overridden) This is the location where the orderer will listen, for example, `0.0.0.0`. Note: unlike the peer, the `orderer.yaml` separates the address and the port, hence the [General.ListenPort](#general-listenport) parameter.
+
+## General.ListenPort
+
+```
+# Listen port: The port on which to bind to listen.
+ListenPort: 7050
+```
+
+* **`ListenPort`**: (default value should be overridden) This is the port that the orderer listens on.
+
+## General.TLS
+
+```
+Enabled: false
+# PrivateKey governs the file location of the private key of the TLS certificate.
+PrivateKey: tls/server.key
+# Certificate governs the file location of the server TLS certificate.
+Certificate: tls/server.crt
+RootCAs:
+  - tls/ca.crt
+ClientAuthRequired: false
+ClientRootCAs:
+```
+
+* **`Enabled`**: (default value should be overridden) In a production network, you should be using TLS-secured communications. This value should be `true`.
+* **`PrivateKey`**: (default value should be overridden). Provide the path to and file name of the private key generated by your TLS CA for this node.
+* **`Certificate`**: (default value should be overridden) Provide the path to and filename of the public certificate (also known as the sign certificate) generated by your TLS CA for this node.
+* **`RootCAs`**: (should be commented out) This parameter needs to be unset for normal use. It can be used to augment the orderer TLS certificate in the channel configuration orderer `MSPDir/tlscacerts` folder.
+* **`ClientAuthRequired`**: (Mutual TLS only) Setting this value to “true” will enable mutual TLS on your network, and must be done for the entire network, not just one node.
+* **`ClientRootCAs`**: (Mutual TLS only)  Can be left blank if mutual TLS is disabled. If mutual TLS is enabled, this is the path to the root certificate of the TLS CA.
```

Comment:
Jason made the same point about this one in the google doc actually, which I missed during my initial review... the set of org tls ca certs from the channel config will be used to verify client connections. You can add additional tls ca certs in ClientRootCAs to augment that list. Again, it would be rare.